docs/develop.rst

```diff
@@ -109,21 +109,20 @@ The method defines the follwing things:
 - ``plonecli_alias``: a short name alias which will be used by plonecli
 - ``depend_on``: an optional global parent template
 
-We use here global unique template names which have the ``plone_`` prefix. Thats because other bobtemplate packages might register templates two and we want avoit naming clashes.
-
+We use here global unique template names which have the ``plone_`` prefix. Thats because other bobtemplate packages might register templates too and we want avoit naming clashes.
```

Comment:
- Put second sentence on its own line, per the style guide.
- s/global/globally
- s/Thats/That is
- s/we want avoit naming clashes/we want to avoid name clashes